src/Fist/Http/tests/RequestTest.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+use Fist\Http\Request;
+use Fist\Testing\TestCase;
+
+class RequestTest extends TestCase
```

Comment:
__[code_sniffer]__
* Each class must be in a namespace of at least one level (a top-level vendor name)

<!-- Comment by [SideCI](https://sideci.com) -->